src/cider/nrepl/middleware/debug.clj

```diff
@@ -207,12 +207,14 @@ this map (identified by a key), and will `dissoc` it afterwards."}
 
 (defn- eval-with-locals
   "`eval` form wrapped in a let of the locals map.
-  If an exception is thrown, it is caught and sent to the client, and
-  this function returns nil."
-  [form locals]
+  If an exception is thrown, it is caught and sent to the client, and this
+  function returns nil. `extras` is a metadata map received from `break'."
+  [form extras]
```

Comment:
I'd name `extras` something else - without reading its docstring it was hard for me to figure out what it is. `break-state` perhaps?